lib/grp.gd

```diff
@@ -711,12 +714,11 @@ InstallTrueMethod( IsSimpleGroup, IsSporadicSimpleGroup );
 ##  <#/GAPDoc>
 ##
 DeclareProperty( "IsAlmostSimpleGroup", IsGroup );
-InstallTrueMethod( IsGroup, IsAlmostSimpleGroup );
+InstallTrueMethod( IsGroup and IsNonTrivial, IsAlmostSimpleGroup );
 
-# a simple group is almost simple
+# nonabelian simples group are almost simple
```

Comment:
```suggestion
# nonabelian simple group are almost simple
```

Reply:
GitHub has assumed that my neutral review was an approving reviewing. Wouldn't be terrible to merge with this typo, but why not fix it ;)

Reply:
Oops, good catch, thanks!